VTEX - Budgets API.json

```diff
@@ -0,0 +1,5235 @@
+                }
+              }
+            }
+          },
+          "404": {
+            "description": "Not Found",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/responseError"
+                },
+                "example": {
+                  "error": "Resource not found."
+                }
+              }
+            }
+          },
+          "409": {
+            "description": "Conflict",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/responseError"
+                },
+                "example": {
+                  "error": "Cannot delete resource due to existing dependencies."
+                }
+              }
+            }
+          },
+          "500": {
+            "description": "Internal Server Error",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/responseError"
+                },
+                "example": {
+                  "error": "An internal server error occurred."
+                }
+              }
+            }
+          }
+        },
+        "security": [
+          {
+            "VtexIdclientAutCookie": []
+          }
+        ]
+      }
+    },
+    "/api/budgets/{contextType}/{contextId}/{budgetId}/allocations/{allocationId}/status": {
+      "put": {
+        "tags": [
+          "Allocations"
+        ],
+        "summary": "Change allocation status",
+        "description": "This endpoint updates the status of a specific allocation within a budget.\n\n## Permissions\n\nThis endpoint does not require [License Manager resources](https://help.vtex.com/en/tutorial/license-manager-resources--3q6ztrC8YynQf6rdc6euk3).",
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/Content-Type"
+          },
+          {
+            "$ref": "#/components/parameters/contextType"
+          },
+          {
+            "$ref": "#/components/parameters/contextId"
+          },
+          {
+            "$ref": "#/components/parameters/budgetId"
+          },
+          {
+            "$ref": "#/components/parameters/allocationId"
+          }
+        ],
+        "requestBody": {
+          "required": true,
+          "content": {
+            "application/json": {
+              "schema": {
+                "type": "object",
+                "description": "Request body for changing allocation status.",
+                "required": [
+                  "status"
+                ],
+                "properties": {
+                  "status": {
+                    "type": "string",
+                    "description": "The new status value to apply to allocation (e.g., `ACTIVE` or `INACTIVE` ).",
+                    "enum": [
+                      "ACTIVE",
+                      "INACTIVE"
```

Comment:
I believe it's worth it mentioning that the `DELETED` status is not applicable to the context.